### PR TITLE
Development

### PR DESCRIPTION
**Pull request by mvassari**
Trying to push the RAC-CCNY Internship Program Overview to live site

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Docs template
+# RAC-CCNY Internship Program Overview
 A template for docs.rockarch.org documentation repositories. Include a short description of the documentation.
```
Comment (daremiyo) [inline README.md]: Need to include description here

**Review by daremiyo — CHANGES_REQUESTED** [review]
Looks good! I requested a small change in the readme

File: README.md (hunk 1 above)
Comment (mvassari) [reply README.md]: Updated Readme

**Review by mvassari — COMMENTED** [review]
Hi @daremiyo did my description go through?

**Review by mvassari — COMMENTED** [review]
Hi @daremiyo did my description go through?

**Review by daremiyo — APPROVED** [review]

**Pull request merged**